test/test.combokeys.js

```diff
@@ -184,15 +184,26 @@ describe('combokeys.bind', function () {
       var combokeys = new Combokeys(element)
       combokeys.bind('command+s', spy)
 
+      if (Event.prototype.preventDefault) {
+        var preventDefaultSpy = sinon.spy(Event.prototype, 'preventDefault')
+      }
+      if (Event.prototype.stopPropagation) {
+        var stopPropagationSpy = sinon.spy(Event.prototype, 'stopPropagation')
+      }
       KeyEvent.simulate('S'.charCodeAt(0), 83, ['meta'], element)
 
       assert.strictEqual(spy.callCount, 1, 'callback should fire')
       assert.instanceOf(spy.args[0][0], Event, 'first argument should be Event')
       var event = spy.args[0][0]
       if (event.preventDefault) {
-        assert.isTrue(event.defaultPrevented)
+        assert.isTrue(preventDefaultSpy.calledOnce, 'default action was cancelled')
+      } else {
+        assert.isFalse(event.returnValue, 'default is prevented')
+      }
+      if (event.stopPropagation) {
+        assert.isTrue(stopPropagationSpy.calledOnce, 'propagation was cancelled')
```

Comment:
I can't see any other way to check whether propagation was stopped.

Apparently, [this](http://www.w3.org/TR/2003/WD-DOM-Level-3-Events-20030331/events.html#Events-Event-isPropagationStopped) is not implemented.